### PR TITLE
 layout category

### PR DESCRIPTION
**Pull request by ghost**
## Description
lay out category
## Reference Requirements
- Task ID: TI-1

## Reference Isuees
#{issue_id}

## Migrations
NO

## Impacted Areas in Application
List general components of the application that this PR will affect:

-

### CODE REVIEW

File: resources/views/user/layout/sidebar.blade.php [hunk 1]
```diff
@@ -8,8 +8,58 @@
...
+              <div class="list-group-item is-child">
+                <a href="">Tivi Màn Hình Cong</a>
+              </div> 
+              <div class="list-group-item">
+                  <a href="" >
+                      Android TV Box, Smart Box
+                  </a>                    
+              </div>
+            </div> -->
+            <nav>
```
Comment (at-thaitran) [inline resources/views/user/layout/sidebar.blade.php]: tai sao phai cmt đoạn này

File: resources/views/user/layout/sidebar.blade.php [hunk 2]
```diff
@@ -38,7 +88,7 @@
                 <input type="checkbox" id="jtv8" name="jtvc">
                 <label for="jtv8"> <span class="button"></span> Best selling<span class="count"></span> </label>
               </li>
-            </ul>
```
Comment (at-thaitran) [inline resources/views/user/layout/sidebar.blade.php]: xóa cmt

File: public/css/user/add.css [hunk 3]
```diff
@@ -4,3 +4,21 @@
...
+    padding-left: 10px;
+}
+
+ul.slideContent {
+  display: none;
+}
+
+a.header {
+    color: #4caf50 ;
+}
```
Comment (at-thaitran) [inline public/css/user/add.css]: css dùng 2 space thôi

**Comment by at-thaitran** [comment]
LGTM

**Comment by at-vietpham** [comment]
LGTM 🍶

**Pull request merged**